.github/workflows/CI-unix.yml

```
@@ -36,7 +36,7 @@ jobs:
     env:
       ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

```suggestion
      - uses: actions/checkout@v3
```
Seems like we got a soft NAK from CI on this update, for unclear reasons: https://github.com/libuv/libuv/actions/runs/10151174740/job/28069896719?pr=4474